CITATION.cff

```diff
@@ -0,0 +1,42 @@
+abstract: "EICrecon is the EIC online/offline reconstruction software."
+authors: # in the order of https://zenodo.org/records/15880663/export/cff
+- given-names: The ePIC Collaboration
+- family-names: Wouter Deconinck
+- family-names: David Lawrence
+- family-names: Dmitry Romanov
+- family-names: Dmitry Kalinkin
+- family-names: Christopher Dilks
+- family-names: Thomas Britton
+- family-names: Simon Gardner
+- family-names: Tooba Ali
+- family-names: Nathan Brei
+- family-names: Xinxin (Cissie) Mei
+- family-names: Derek M Anderson
+- family-names: Barak Schmookler
+- family-names: Ryan Milton
+- family-names: Sebouh Paul
+- family-names: Alex Jentsch
+- family-names: John Lajoie
+- family-names: Sakib Rahman
+- family-names: Joseph (Joe) Osborn
+- family-names: Shyam Kumar
+- family-names: Tommy Chun Yuen Tsang
+- family-names: Zhongling Ji
+- family-names: Matt Posik
+- family-names: Peter Steinberg
+- family-names: Minho Kim
+- family-names: Daniel Brandenburg
+- family-names: Dhevan Gangadharan
```

Comment:
How do we keep this up to date?

Reply:
I was wondering about that too. It's related to the problem with the versions. If we could automate our version release workflow and include a download of a new CITATION.cff, then we can get away from this. Otherwise, I am just tempted to keep it as is, and replace it maybe on demand (we can add a comment line to the file) or on schedule. We could probably even do a scheduled job that downloads the file, checks for diffs, and opens a PR with the necessary changes. But that's all work...

Reply:
That's until Zenodo will start feeding you your own cff?